components/Cards/src/MDCCardCollectionCell.h

```diff
@@ -0,0 +1,158 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+#import "MDCCard.h"
+
+typedef NS_ENUM(NSInteger, MDCCardCellState) {
+  /** The visual state when the cell is in its normal state. */
+  MDCCardCellStateNormal = 0,
+
+  /** The visual state when the cell is in its highlighted state. */
+  MDCCardCellStateHighlighted,
+  
+  /** The visual state when the cell has been selected. */
+  MDCCardCellStateSelected
+};
+
+@interface MDCCardCollectionCell : UICollectionViewCell
+
+/**
+ When selecting is set to YES, a tap on a cell will trigger a visual change between selected
+ and unselected. When it is set to NO, a tap will trigger a normal tap (rather than trigger
+ different visual selection states on the card).
+ Default is set to NO.
+ */
+@property(nonatomic, assign, getter=isSelecting) BOOL selecting;
+
+/**
+ The corner radius for the card
+ Default is set to 4.
+ */
+@property(nonatomic, assign) CGFloat cornerRadius UI_APPEARANCE_SELECTOR;
+
+/**
+ The inkView for the card that is initiated on tap
+ */
+@property(nonatomic, readonly, strong, nonnull) MDCInkView *inkView;
+
+/**
+ Sets the shadow elevation for an MDCCardViewState state
+
+ @param shadowElevation The shadow elevation
+ @param state MDCCardCellState the card state
+ */
+- (void)setShadowElevation:(MDCShadowElevation)shadowElevation forState:(MDCCardCellState)state
+    UI_APPEARANCE_SELECTOR;
+
+/**
+ Returns the shadow elevation for an MDCCardViewState state
+
+ If no elevation has been set for a state, the value for MDCCardCellStateNormal will be returned.
+ Default value for MDCCardCellStateNormal is 1
+ Default value for MDCCardCellStateHighlighted is 8
+ Default value for MDCCardCellStateSelected is 8
+
+ @param state MDCCardCellStateNormal the card state
+ @return The shadow elevation for the requested state.
+ */
+- (MDCShadowElevation)shadowElevationForState:(MDCCardCellState)state;
+
+/**
+ Sets the border width for an MDCCardViewState state
+
+ @param borderWidth The border width
+ @param state MDCCardCellState the card state
+ */
+- (void)setBorderWidth:(CGFloat)borderWidth forState:(MDCCardCellState)state
+    UI_APPEARANCE_SELECTOR;
+
+/**
+ Returns the border width for an MDCCardCellState state
+
+ If no border width has been set for a state, the value for MDCCardCellStateNormal will be returned.
+ Default value for MDCCardCellStateNormal is 0
```

Comment:
I think you can simplify the defaults just keeping this line (`.normal`) since in the line above you mention that the value of `.normal` is used if no explicit value is set for other states.